libs/git_wrapper/Git_wrapper.ml

```diff
@@ -179,3 +179,13 @@ let commit cwd msg =
   match Bos.OS.Cmd.run_status cmd with
   | Ok (`Exited 0) -> ()
   | _ -> raise (Error "Error running git commit")
+
+let get_project_url () =
+  let cmd = Bos.Cmd.(v "git" % "ls-remote" % "--get-url") in
+  let out = Bos.OS.Cmd.run_out cmd in
+  match Bos.OS.Cmd.out_string ~trim:true out with
+  | Ok (url, _) -> Some url
+  | Error _ ->
+      File.find_first_match_with_whole_line (Fpath.v ".git/config") ".com"
+(* TODO(dinosaure): this function is pretty weak. We probably should handle that
```

Comment:
What do you mean by weak? What is the issue?

Reply:
I mean, for my point of view, depending on what we can have into the `.git/config`, the result can be wrong. For instance, in Robur, we use `https://git.robur.coop/` which will not be recognized by this function. I'm not sure why we need to `find_first_match_with_whole_line` here when `git ls-remote --get-url` seems right and enough.

Reply:
ah it's because of the .com? Yes, seems bad, but maybe you can add a comment with your example (and saying you wrote this because that's how it was done in pysemgrep)